mod-source-record-manager-server/src/test/java/org/folio/dao/JobExecutionDaoImplTest.java

```diff
@@ -104,11 +147,193 @@ private Future<RowSet<Row>> createProgressForJobExecutions(List<JobExecution> jo
           .withCurrentlySucceeded(random.nextInt(10))
           .withCurrentlyFailed(random.nextInt(10))
           .withTotal(random.nextInt(10));
-
         future = future.compose(ar -> jobExecutionProgressDao.save(progress, params.getTenantId()));
       }
     }
     return future;
   }
 
+  @Test
+  public void shouldDeleteRecordsMarkedForJobExecutionsDeletion(TestContext context) {
+    /*
+      In this test we setup job execution that finishes 3 days ago, after this job execution has been soft deleted.
+      Periodic job should hard delete it, because it checks soft deleted jobs older than 2 days.
+    */
+    Async async = context.async();
+
+    Instant minusThreeDays = LocalDate.now().minusDays(3).atStartOfDay().atZone(ZoneId.systemDefault()).toInstant();
```

Comment:
I think all the timezones in the test should be set to UTC specifically. This will allow developer machines to have accurate tests/ data in the db since most will not be configured for UTC.

Reply:
Missed it during development, updated to UTC, thanks